projects/demo/src/pages/documentation/mask-expression/mask-expression.template.html

```diff
@@ -98,55 +103,72 @@ <h2>Types of mask expression</h2>
 
                 <p>
-                    This mask expression forbids something excepts digits or
-                    input more than four digits.
+                    This mask expression forbids anything excepts digits and
+                    limits length of the value to 5 characters.
                 </p>
-                <p class="tui-space_bottom-0">
-                    Also, it manages user's interactions with fixed character.
-                </p>
-                <p class="tui-space_top-0 tui-space_bottom-0">
-                    For example, user can press just four digits
+
+                <p>Also, it manages user interactions with fixed character.</p>
+
+                <p>
+                    For example, user can just type four digits
                     <code>1159</code>
-                    and the input's value becomes
+                    and the value becomes
                     <code>11:59</code>
                 </p>
-                <p class="tui-space_top-0">
-                    Another example, if caret position is after colon and user
-                    presses
+
+                <p>
+                    Another example, if caret position is after the colon and
+                    user presses
                     <kbd>Backspace</kbd>
                     , the input's value will not change but caret will be moved
-                    to the left from colon.
+                    to the left of the colon.
                 </p>
             </li>
-        </ul>
-    </section>
+            <li class="tui-list__item">
+                <strong>Dynamic mask expression</strong>
 
-    <section class="tui-space_top-12">
-        <h2>Dynamic mask expression</h2>
+                <p class="tui-space_top-0">
+                    <code>mask</code>
+                    parameter can also accepts function which generates mask
+                    expression. This function will be called
+                    <strong>every time before</strong>
+                    input changes to generate a new version of mask expression.
+                </p>
 
-        <p>
-            <code>mask</code>
-            -parameter of Maskito's options can also accepts function which
-            generates mask expression. This function will be called
-            <strong>every time before</strong>
-            any input's changes to generate new version of mask expression.
-        </p>
+                <tui-notification
+                    status="info"
+                    class="tui-space_bottom-3"
+                >
+                    An "element state" object with raw value and current
+                    selection is passed as an argument to the function. See more
+                    about element state in
+                    <a
+                        tuiLink
+                        [routerLink]="processorsDocPage"
+                    >
+                        Processors
```

Comment:
Probably, we should create the separate topic "Element state" inside "Core concepts" 🤔 

I will create issue for it now

Reply:
- https://github.com/Tinkoff/maskito/issues/152